[WIP] Formatting

CodaFi opened the pull request:
Leaving this here so we don't accidentally work on the same thing.

CodaFi committed:
Update dependencies

CodaFi committed:
Space variables

CodaFi committed:
Format parameters

pthariensflame commented:
:)

CodaFi committed:
Get some autoclosure'd parameters too

CodaFi committed:
Generic parameter spacing

CodaFi committed:
Document NSFileHandle extension

CodaFi committed:
Good god, my eyes

CodaFi committed:
Fix accidental newline

CodaFi committed:
Move some more operators

CodaFi committed:
Final cleanup

CodaFi commented:
Ready for review :eyes: .

CodaFi commented on Aquifer/NSFileHandleExt.m:
```diff
@@ -15,129 +15,140 @@
 #define BUFFER_SIZE 256
 
 in_addr_t getipaddr(const char *host) {
-    /* let's look it up */
```
Good god, what is this stuff?  I'm going to see what I can do about this weirdness.

pthariensflame replied:
Please do!  :O

CodaFi committed:
Use asserts

CodaFi committed:
Remove public variables

pthariensflame commented:
This looks good to me so far.  I may end up going through the codebase after this is merged just to catch any subtle formatting errors, but for current purposes this is fine.

CodaFi committed:
DONT MODIFY WHAT YOU DONT OWN DONT MODIFY WHAT YOU DONT OWN DONT

MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT
OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU
DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT
YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY
WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN MODIFY WHAT YOU DONT OWN
MODIFY WHAT YOU DONT OWN

CodaFi commented:
I'm done for now, then.

Merge when ready.

pthariensflame commented:
:boat:, then!
